### PR TITLE
use inspektor-gadget instead of falco

### PR DESCRIPTION
**Pull request by matthyx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ sudo SNIFFER_CONFIG=./configuration/SnifferConfigurationFile.json ./sniffer
 
 ## Limitations:
 1. This feature is using EBPF technology that is implemented only on linux.
-2. the linux kernel version that supported it 4.14
+2. the linux kernel version that supported it 5.4 and above.
```
Comment (rcohencyberarmor) [inline README.md:17]: kernel version need to be 5.10

Comment (rcohencyberarmor) [reply README.md:17]: https://github.com/inspektor-gadget/inspektor-gadget#kernel-requirements

File: internal/validator/validator.go [hunk 2]
```diff
@@ -2,20 +2,21 @@ package validator
...
+	"github.com/facette/natsort"
 	"github.com/kubescape/go-logger"
 	"github.com/kubescape/go-logger/helpers"
+	"golang.org/x/sys/unix"
 )
 
 const (
-	FalcoEBPFEngineMinKernelVersionSupport     = "4.14"
-	KubescapeEBPFEngineMinKernelVersionSupport = "5.1"
+	KubescapeEBPFEngineMinKernelVersionSupport = "5.4"
```
Comment (rcohencyberarmor) [inline internal/validator/validator.go:17]: here as well: 5.10
https://github.com/inspektor-gadget/inspektor-gadget#kernel-requirements

**Review by rcohencyberarmor — CHANGES_REQUESTED** [review]

**Review by rcohencyberarmor — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (matthyx) [reply README.md:17]: https://github.com/inspektor-gadget/inspektor-gadget/blob/main/docs/requirements.md

**Pull request merged**